base/include/ColorConversionXForm.h

```diff
@@ -0,0 +1,60 @@
+#pragma once
+#include "FrameMetadata.h"
+#include "Module.h"
+#include "AbsColorConversionFactory.h"
+
+class ColorConversionProps : public ModuleProps
+{
+public:
+	enum ConversionType
+	{
+		RGB_2_MONO = 0,
```

Comment:
The numeric '2' is conusing... it should br "TO"

Reply:
Done, changed it!!